client/network/src/block_request_handler.rs

```diff
@@ -40,13 +40,13 @@ use std::hash::{Hasher, Hash};
 const LOG_TARGET: &str = "sync";
 const MAX_BLOCKS_IN_RESPONSE: usize = 128;
 const MAX_BODY_BYTES: usize = 8 * 1024 * 1024;
-const MAX_NUMBER_OF_SAME_REQUESTS_PER_PEER: usize = 2;
+const MAX_NUMBER_OF_SAME_REQUESTS_PER_PEER: usize = 6;
```

Comment:
Otherwise tests don’t pass. Should I try to fix tests instead ?

Reply:
Because your changes break them.